test/e2e/cluster_extension_admission_test.go

```diff
@@ -0,0 +1,75 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	ocv1alpha1 "github.com/operator-framework/operator-controller/api/v1alpha1"
+)
+
+func TestClusterExtensionPackageUniqueness(t *testing.T) {
```

Comment:
Feels a bit like we are testing upstream Kubernetes... Is this test worth having?

Reply:
On the other hand - I got CEL expression wrong at first while working on #774: I blocked edits :) Given that I think it makes sense.